src/components/map/tools/SplitLinkTool.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import ToolbarTool from '~/enums/toolbarTool';
+import EventManager from '~/util/EventManager';
+import ConfirmStore from '~/stores/confirmStore';
+import NetworkStore, { MapLayer } from '~/stores/networkStore';
+import NodeService from '~/services/nodeService';
+import ErrorStore from '~/stores/errorStore';
+import LinkStore from '~/stores/linkStore';
+import ToolbarStore from '~/stores/toolbarStore';
+import NodeType from '~/enums/nodeType';
+import NodeHelper from '~/util/nodeHelper';
+import RouteBuilder from '~/routing/routeBuilder';
+import navigator from '~/routing/navigator';
+import SubSites from '~/routing/subSites';
+import BaseTool from './BaseTool';
+import * as s from './splitLinkTool.scss';
+
+/**
+ * Tool for splitting link
```

Comment:
Maybe this comment isn't needed. I agree this comment unnecessary in some other tools as well